nuxt.config.ts

```diff
@@ -158,17 +158,7 @@ export default defineNuxtConfig({
     extractCSS: true,
   },
 
-  modules: ['@pinia/nuxt'],
-
-  buildModules: [
```

Comment:
Removed because it was deprecated.
Reference: https://nuxt.com/docs/guide/concepts/modules#add-nuxt-modules